lib/apiservers/engine/backends/image.go

```diff
@@ -81,8 +84,17 @@ func (i *Image) Images(filterArgs string, filter string, all bool) ([]*types.Ima
 	return result, nil
 }
 
+// Docker Inspect.  LookupImage looks up an image by name and returns it as an
+// ImageInspect structure.
 func (i *Image) LookupImage(name string) (*types.ImageInspect, error) {
-	return nil, fmt.Errorf("%s does not implement image.LookupImage", i.ProductName)
+	defer trace.End(trace.Begin("LookupImage (docker inspect)"))
```

Comment:
Could just pass `name` to trace being - allows us to differentiate concurrent requests easily.